### PR TITLE
add settlement frequency link complete

### PR DESCRIPTION
**Pull request by Justicea83**
- [ ] HOLD on https://github.com/Expensify/Web-Expensify/pull/33250

### Details

This PR needs this https://github.com/Expensify/Web-Expensify/pull/33250 to be merged before it works.

### Fixed Issues

$https://github.com/Expensify/Expensify/issues/199498

### Tests
1. Click on your profile picture
2. Go to your workspace
3. Go to issue cards
4. Click on Settlement Frequency
5. It should redirect you to olddot settings tab with this picture

Screenshot 2022-03-09 at 9 21 03 AM

For example:
1. Click on the text input to bring it into focus
7. Upload an image via copy paste
8. Verify a modal appears displaying a preview of that image
--->

- [ ] Verify that no errors appear in the JS console

#### Contributor (PR Author) Checklist
- [ ] I verified the PR has a small number of commits behind `main`
- [ ] I linked the correct issue in the `### Fixed Issues` section above
- [ ] I wrote clear testing steps that cover the changes made in this PR
    - [ ] I clearly indicated the environment tests should be run in (Staging vs Production)
- [ ] I wrote testing steps that cover success & fail scenarios (if applicable)
- [ ] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [ ] I ran the tests & veryfy they passed on:
    - [ ] iOS / native
    - [ ] Android / native
    - [ ] iOS / Safari
    - [ ] Android / Chrome
    - [ ] MacOS / Chrome
    - [ ] MacOS / Desktop
- [ ] I verified there are no console errors related to changes in this PR
- [ ] I followed proper code patterns (see [Reviewing the code](https://github.com/Expensify/App/blob/main/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [ ] I added comments when the code was not self explanatory
    - [ ] I put all copy / text shown in the product in all `src/languages/*` files (if applicable)
    - [ ] I followed proper naming convention for platform-specific files (if applicable)
    - [ ] I followed style guidelines (in [`Styling.md`](https://github.com/Expensify/App/blob/main/STYLING.md)) for all style edits I made
    - [ ] I followed the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/STYLE.md#jsdocs))
- [ ] I followed the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/PR_REVIEW_GUIDELINES.md)
- [ ] I tested other components that can be impacted by my changes (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` are working as expected)
- [ ] I corroborated the UI performance was not affected (the performance is the same than `main` branch)
- [ ] If I created a new component I verified that a similar component doesn't exist in the codebase

#### PR Reviewer Checklist
- [ ] I verified the PR has a small number of commits behind `main`
- [ ] I verified the correct issue is linked in the `### Fixed Issues` section above
- [ ] I verified testing steps are clear and they cover the changes made in this PR
    - [ ] I verified the testing environment is mentioned in the test steps
- [ ] I verified testing steps cover success & fail scenarios (if applicable)
- [ ] I checked that screenshots or videos are included for tests on [all platforms](https://github.com/Expensify/App/blob/main/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [ ] I verified tests pass on **all platforms** & I tested again on:
    - [ ] iOS / native
    - [ ] Android / native
    - [ ] iOS / Safari
    - [ ] Android / Chrome
    - [ ] MacOS / Chrome
    - [ ] MacOS / Desktop
- [ ] I verified there are no console errors related to changes in this PR
- [ ] I verified proper code patterns were followed (see [Reviewing the code](https://github.com/Expensify/App/blob/main/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [ ] I verified comments were added when the code was not self explanatory
    - [ ] I verified any copy / text shown in the product was added in all `src/languages/*` files (if applicable)
    - [ ] I verified proper naming convention for platform-specific files was followed (if applicable)
    - [ ] I verified [style guidelines](https://github.com/Expensify/App/blob/main/STYLING.md) were followed
    - [ ] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/STYLE.md#jsdocs)) were followed
- [ ] I verified that this PR follows the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/PR_REVIEW_GUIDELINES.md)
- [ ] I verified other components are not impacted by changes in this PR (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` are working as expected)
- [ ] I verified the UI performance was not affected (the performance is the same than `main` branch)
- [ ] If a new component is created I verified that a similar component doesn't exist in the codebase

### QA Steps

- [ ] Verify that no errors appear in the JS console

### CODE REVIEW

File: src/pages/workspace/card/WorkspaceCardVBAWithECardView.js
Comment (madmax330) [inline src/pages/workspace/card/WorkspaceCardVBAWithECardView.js:36]: This needs to be behind the `monthlySettlement` beta

Comment (Justicea83) [reply src/pages/workspace/card/WorkspaceCardVBAWithECardView.js:36]: Have been researching and found an SO but all the info is about how to implement Beta in Web-Expensify
You know how i can do same in newdot?

Comment (madmax330) [reply src/pages/workspace/card/WorkspaceCardVBAWithECardView.js:36]: I'm not sure myself, I don't really work in newDot. But if you search the codebase you should be able to find some examples. Alternatively you can ask in #eng-chat

Comment (nkuoch) [reply src/pages/workspace/card/WorkspaceCardVBAWithECardView.js:36]: check `Permissions.canUseWallet(props.betas)` and you can do something similar

**Comment by Justicea83** [comment]
updated and waiting for another review

**Comment by nkuoch** [comment]
hmm I think you hid it behind the wrong beta.

File: src/pages/workspace/card/WorkspaceCardVBAWithECardView.js
Comment (madmax330) [inline src/pages/workspace/card/WorkspaceCardVBAWithECardView.js:34]: This isn't the right beta. It should be `monthlySettlements` but the beta isn't on newDot. So you have to add that in

**Comment by Justicea83** [comment]
updated and awaiting another review

File: src/libs/Permissions.js
Comment (madmax330) [inline src/libs/Permissions.js:82]: You need to export this

File: src/pages/workspace/card/WorkspaceCardVBAWithECardView.js
Comment (Justicea83) [reply src/pages/workspace/card/WorkspaceCardVBAWithECardView.js]: updated and awaiting another review

**Comment by Justicea83** [comment]
updated and awaiting another review

**Review by nkuoch — APPROVED** [review]

**Comment by Justicea83** [comment]
Gentle bump @madmax330

**Pull request merged**
